RepoCleanup/Functions/SetupNewServiceOwnerFunction.cs

```diff
@@ -42,21 +41,33 @@ public static async Task Run()
             {
                 Console.WriteLine($"Created all default teams for {org.Fullname}");
             }
-            
+
             // Create default datamodels repository
+            var isDatamodelRepoCreated = await CreateRepo(giteaService, org, "datamodels", true);
+
+            // Create default content repository
+            var isContentRepoCreated = await CreateRepo(giteaService, org, "content", false);
```

Comment:
Now that we have these helper methods, the comments are a little redundant.
But we should still keep one comment, to follow the pattern of the class.

```suggestion
            // Create default repositories
            var isDatamodelRepoCreated = await CreateRepo(giteaService, org, "datamodels", true);
            var isContentRepoCreated = await CreateRepo(giteaService, org, "content", false);
```